aggregate/clean_aggregated.py

```diff
@@ -4,6 +4,16 @@
 
 races = ["anh", "bnh", "hsp", "wnh"]
 
+reorder_mapper = {
```

Comment:
Makes sense to define as constant and load from helper module. Only comment is that it might be nicer to have a variable name that specifies that this mapper is for dcp population spreadsheets? Not necessary though